ci/jenkins/test.sh

```diff
@@ -522,8 +522,6 @@ function  build_and_deliver_antrea_windows_and_linux_containerd_images {
     kubectl delete -f ${WORKDIR}/antrea.yml --ignore-not-found=true || true
 
     prepare_env
-    ${CLEAN_STALE_IMAGES_CONTAINERD}
```

Comment:
Please update the commit message, including why we remove these commands on the new testbed.

Reply:
Reminder @NamanAg30